tests/physics/helicity_formalism/angular_distributions/d1_to_d0_pi_pi_decay/__init__.py

```diff
@@ -0,0 +1 @@
+"""Required to set mypy options for the tests folder."""
```

Comment:
Why put these tests in different folders (see comments about committing the `model.yml` files).

Reply:
ac910b9